bindings/generate.sh

```diff
@@ -44,17 +44,17 @@ generate() {
     rm -r $PKG
     mkdir $PKG
 
-    # Generate bindings
-    $ABIGEN --pkg $PKG --sol ../contracts/contracts/$FILE.sol --out $PKG/$FILE.go --solc $SOLC
-
-    # Generate binary runtime
-    $SOLC --bin-runtime --optimize --allow-paths ../contracts/vendor, ../contracts/contracts/$FILE.sol -o $PKG/
+    # Compile and generate binary runtime.
+    $SOLC --abi --bin --bin-runtime --optimize --allow-paths ../contracts/vendor, ../contracts/contracts/$FILE.sol -o $PKG/
     BIN_RUNTIME=`cat ${PKG}/${CONTRACT}.bin-runtime`
     OUT_FILE="$PKG/${CONTRACT}BinRuntime.go"
-    echo "package $PKG // import \"github.com/perun-network/perun-eth-backend/bindings/$PKG\"" > $OUT_FILE
+    echo "package $PKG // import \"perun.network/go-perun/backend/ethereum/bindings/$PKG\"" > $OUT_FILE
```

Comment:
This looks wrong. We should test bindings generation in the CI.

Reply:
Fixed